lib/formfields/customer/domains/formfield.domains_edit.php

```diff
@@ -101,7 +101,12 @@
 						'label' => $lng['domain']['openbasedirpath'],
 						'type' => 'select',
 						'select_var' => $openbasedir
-					)
+					),
+                                        'authcode' => array(
```

Comment:
for the customer, i would hide this, if there is no authcode, e.g.
`visible => (!empty($result['authcode']))`